packages/api/src/router/user.ts

```diff
@@ -130,6 +134,125 @@ export const userRouter = createTRPCRouter({
       }
     }),
 
+  payCoinsByUserId: publicProcedure
+    .input(
+      z.object({
+        receiver: z.object({
+          id: z.string(),
+          username: z.string(),
+          avatar: z.nullable(z.string()),
+          discriminator: z.string(),
+        }),
+        sender: z.object({
+          id: z.string(),
+          username: z.string(),
+          avatar: z.nullable(z.string()),
+          discriminator: z.string(),
+        }),
+        coins: z.number(),
+      }),
+    )
+    .mutation(async ({ ctx, input }) => {
+      try {
+        /**
+         * This function set a temp thumbnail for the user
+         */
+        const tempThumbnail = setTempThumbnail(input.receiver);
```

Comment:
Yo lo llamaria mejor: 

```suggestion
        const thumbnailUrl = setThumbnailUrl(input.receiver);
```

A la final lo que te esta devolviendo la function es una URL